inc/Plugin.php

```diff
@@ -96,13 +96,37 @@ public function register_hooks(): void {
 						$locale  = GP_Locales::by_slug( $translation_set->locale );
 						$project = GP::$project->get( $translation_set->project_id );
 
-						return sprintf(
+						/**
+						 * Filters whether a Slack notification for translation updates from GitHub should be sent.
+						 *
+						 * @since 2.0.3
+						 *
+						 * @param bool               $send_message    Whether to send a notification or not. Default true.
+						 * @param GP_Translation_Set $translation_set Translation set the ZIP is for.
+						 * @param GP_Project         $project         The GlotPress project that was updated.
+						 */
+						$send_message = apply_filters( 'traduttore_zip_generated_send_notification', true, $translation_set, $project );
```

Comment:
Don't we want to separate the "namespace" and the action with a dot? `traduttore.zip_generated_send_notification`

Reply:
I hesitated to do so because the two existing actions we have aren't using a dot.

Reply:
We could use the dot in the existing actions and use `do_action_deprecated` for the non-dot version. Then, we could use the dot for the newer ones, too.

Easier to do so now than later :-)